lib/netext/grpcext/stream.go

```diff
@@ -0,0 +1,121 @@
+package grpcext
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/types/dynamicpb"
+)
+
+// Stream is the wrapper around the grpc.ClientStream
+// with some handy methods.
+type Stream struct {
+	method           string
+	methodDescriptor protoreflect.MethodDescriptor
+	raw              grpc.ClientStream
+	marshaler        protojson.MarshalOptions
+}
+
+// ErrCancelled cancelled by client (k6)
+var ErrCancelled = errors.New("cancelled by client (k6)")
```

Comment:
```suggestion
var ErrCanceled = errors.New("cancelled by client (k6)")
```
Should we align with `codes.Canceled`?

Reply:
To keep us consistent with naming. To be clear what I'm suggesting is `s/Cancelled/Canceled/`. But if you think it isn't useful I'm fine to discard the suggestion.

Reply:
I see now, sorry misunderstood the proposal :facepalm: